src/main/resources/db/migration/V2__MyGmailSupportUser.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO "user" (username, password, role, locked, active)
```

Comment:
Won't `active=false`,` locked=true` do the same as your check in the service?
But it's better to keep that check in the service anyway.